compiler/src/typed/printtyp.re

```diff
@@ -730,7 +730,8 @@ and tree_of_argtyplist = (sch, al) =>
         switch (l) {
         | Default(_) =>
           switch (ty.desc) {
-          | TTyConstr(_, [ty], _) => ty
+          | TTyConstr(_, [ty], _)
+          | TTyLink({desc: TTyConstr(_, [ty], _)}) => ty
```

Comment:
You probably want to recurse on this case in the event there are next TTyLinks.

Reply:
Done